argus/config/arestor.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Cloudbase Solutions Srl
```

Comment:
Here we need an extensive commit description on what is arestor and why we use arestor and what are the current limitations.